test/unit/gateways/commerce_hub_test.rb

```diff
@@ -225,6 +225,18 @@ def test_successful_partial_refund
     assert_success response
   end
 
+  def test_successful_credit
+    response = stub_comms do
+      @gateway.credit(@amount, @credit_card, @options)
+    end.check_request do |_endpoint, data, _headers|
+      request = JSON.parse(data)
+      assert_not_nil request['amount']
+      assert_equal request['source']['card']['cardData'], @credit_card.number
+    end.respond_with(successful_credit_response)
+
+    assert_success response
```

Comment:
💬 Comment:

Giving that you are only testing the content of the request, adding that extra `successful_credit_response` and asserting that the response makes no difference.